### PR TITLE
Make getStorageToken easier to follow - name was optional, it should be required

### PR DESCRIPTION
**Pull request by vladsud**

### CODE REVIEW

File: packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts [hunk 1]
```diff
@@ -145,22 +144,24 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
...
+            odspResolvedUrl,
+            this.getStorageToken,
+            true /* throwOnNullToken */,
+        );
+
+        const webSocketTokenFetcher = this.toInstrumentedOdspTokenFetcher(
+            odspLogger,
+            odspResolvedUrl,
+            this.getWebsocketToken,
+            true /* throwOnNullToken */,
```
Comment (jatgarg) [inline packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts:158]: You are changing the behaviour here whether top throw on null token or not while getting web socket token. Please take a look at my other comment realted to this where I guees we don't want to throw on null token for this.

**Comment by jatgarg** [comment]
>             const finalSocketToken = webSocketToken ?? (websocketEndpoint.socketToken || null);

Here if the token is null, then we get that from endpoint.

---
Refers to: packages/drivers/odsp-driver/src/odspDocumentService.ts:276 in 5e72483. [](commit_id = 5e7248329c515f9422acc40fe62f17b3d6f2594f, deletion_comment = False)

**Comment by jatgarg** [comment]
Approving but please take a look at above behaviour.

**Review by jatgarg — APPROVED** [review]

File: packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts (hunk 1 above)
Comment (vladsud) [reply packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts]: Yes, thanks for catching this! It was copy/paste error.

**Pull request merged**